reference-architecture/aws-ansible/playbooks/roles/s3-registry-bucket/tasks/main.yaml

```diff
@@ -1,6 +1,10 @@
 ---
 - name: set the name of the bucket
-  set_fact: s3_bucket="{{ ansible_date_time.epoch }}-openshift-docker-registry"
+  set_fact: s3_bucket="{{ ansible_date_time.epoch }}-openshift-docker-registry-{{ stack_name}}"
```

Comment:
@cooktheryan agree on creating it as part of the cloud formation template. With that in mind, I would probably leverage the stack_id rather than epoch, though.